### PR TITLE
Absolute V1

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/27224)

## Description
Absolute V1 integration.

## Minimum version of Cortex XSOAR
- [x] 6.0.0

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

## Leftovers:
- [x] TPB
- [ ] Documentation

### CODE REVIEW

File: Packs/Absolute/Integrations/Absolute/Absolute.py [hunk 1]
```diff
@@ -0,0 +1,947 @@
...
+    for device in response:
+        parsed_device = {}
+        for key, val in device.items():
+            if val:
+                if key == 'os' and not keep_os_in_list:
+                    parsed_device['osName'] = val.get('name')
+                elif key == 'espInfo':
+                    parsed_device['encryptionStatus'] = val.get('encryptionStatus')
+                else:
+                    parsed_device[key[0].upper() + key[1:]] = val
```
Comment (DeanArbel) [inline Packs/Absolute/Integrations/Absolute/Absolute.py:757]: Please add a comment here about the expected type of `key` - how come can this be a list?

File: Packs/Absolute/Integrations/Absolute/Absolute.py [hunk 2]
```diff
@@ -0,0 +1,947 @@
...
+                              readable_output=human_readable, raw_response=res)
+    else:
+        return CommandResults(readable_output=f"No devices found in {INTEGRATION} for the given filters: {args}")
+
+
+def get_device_command(args, client) -> CommandResults:
+    if not ('device_ids' in args or 'device_names' in args or 'local_ips' in args or 'public_ips' in args):
+        raise DemistoException(
+            f"{INTEGRATION} at least one of the commands args (device_ids, device_names, local_ips, public_ips) "
+            f"must be provided.")
```
Comment (DeanArbel) [inline Packs/Absolute/Integrations/Absolute/Absolute.py:829]: Seems like this could use the generic replacement for `raise_error_on_missing_args`.

**Review by DeanArbel — APPROVED** [review]
Amazing effort. Please see my comments below.

File: Packs/Absolute/Integrations/Absolute/Absolute.py (hunk 1 above)
Comment (abaumgarten) [reply Packs/Absolute/Integrations/Absolute/Absolute.py:762]: Key is not a list, is a string and we take the substring of the first place till the end. The key is actually the devices fields.

**Comment by abaumgarten** [comment]
Assigning @jrosenpanw for doc-review. 
Please reach me when reviewing the yml file (specially the long commands descriptions).
Thanks.

**Comment by ShirleyDenkberg** [comment]
@DeanArbel @guykeller Doc review completed.

**Pull request merged**